addons/rmsmartshape/plugin.gd

```diff
@@ -1033,7 +1033,8 @@ func _input_handle_right_click_press(mb_position: Vector2, grab_threshold: float
 	elif current_mode == MODE.EDIT_EDGE:
 		if on_edge:
 			gui_edge_info_panel.visible = not gui_edge_info_panel.visible
-			gui_edge_info_panel.position = mb_position + Vector2(256, -24)
+			gui_edge_info_panel.position = get_window().get_mouse_position()
```

Comment:
What about the offset?

Reply:
Also, why `get_window().get_mouse_position()` and not `mb_position`?

Reply:
![Without offset](https://user-images.githubusercontent.com/2766569/223283468-52bbbe2a-ccf8-44e9-94b5-d9f083537b37.jpg)
![With offset](https://user-images.githubusercontent.com/2766569/223283470-1ee31e92-9d22-4291-9c96-7c62d105cfb5.jpg)
![Using mb_position instead, without offset](https://user-images.githubusercontent.com/2766569/223284227-6a7b8846-4717-4287-be75-a343e93e4817.jpg)
I noticed that mb_position seems to produce different result depending on how big your editor window is.

Reply:
Using `get_window().get_mouse_position()` the panel behaves more like a proper popup.

Reply:
Ah, then we probably need to adapt more code to use `get_window().get_mouse_position()`, but that's for a later PR. The additional offset really doesn't make sense since it only appears on right click and not permantently like the vertex info panel. So, it's alright I guess.

Reply:
Additional offset also feels like a broken popup window that appears somewhere else than it should, breaking established convention.

Reply:
Yeah